### DISCUSSION TITLE
Can you please confirm whether we install ImageMagick on Microsoft Windows - 11 operating system?

### DISCUSSION THREAD

**Discussion by jagdishtaleleDN** [ImageMagick/ImageMagick#7533] [Help] [answered]
### ImageMagick version

7.1

### Operating system, version and so on

Windows - 11

### Description

Can you please confirm whether we install ImageMagick on Microsoft Windows - 11 operating system?

**Comment by dlemstra** [comment]
I have no idea what you are trying to ask us?

**Reply by jagdishtaleleDN** [reply]
Official document is saying that ImageMagick can installed on Microsoft Windows - 10 operating system but we are planning to install on Microsoft Windows - 11 operating system. Is there any problem to install ImageMagick on Microsoft Windows - 11?

**Reply by dlemstra** [reply, marked as answer]
Where are you seeing this?

And to answer your questions. You can install ImageMagick on Windows 11.

**Reply by jagdishtaleleDN** [reply]
Thank you for your answer. 
You can find information - https://imagemagick.org/script/download.php - 
![image](https://github.com/user-attachments/assets/76ca78ed-b25a-4702-9955-12a87b38ed40)

**Reply by dlemstra** [reply]
That says Windows 7 or newer? Windows 10 and 11 are both newer than Windows 7.
